app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue

```diff
@@ -219,7 +245,15 @@ export default {
       uiFlags: 'contacts/getUIFlags',
       conversationsUiFlags: 'contactConversations/getUIFlags',
       currentUser: 'getCurrentUser',
+      messageSignature: 'getMessageSignature',
     }),
+    sendWithSignature() {
+      const { send_with_signature: isEnabled } = this.uiSettings;
+      return isEnabled;
+    },
+    signatureToApply() {
+      return cleanSignature(this.messageSignature);
```

Comment:
No need to clean signature here, it's already done when applying or removing

Reply:
Done https://github.com/chatwoot/chatwoot/pull/7946/commits/47bb845e58dec10997bde626b018e0ba060b8574